Lists/Exercise/ListOperations/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace ListOperations
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            List<int> numbersList = Console.ReadLine().Split(' ').Select(int.Parse).ToList();
+            string command = Console.ReadLine();
+
+            while (command != "End")
+            {
+                List<string> commandList = command.Split(' ').ToList();
+                if (commandList[0] == "Add")
+                {
+                    numbersList.Add(int.Parse(commandList[1]));
+                }
+                else if (commandList[0] == "Insert")
+                {
+                    if (int.Parse(commandList[2]) < numbersList.Count && int.Parse(commandList[2]) >= 0)
+                    {
+                        numbersList.Insert(int.Parse(commandList[2]), int.Parse(commandList[1]));
```

Comment:
Extract int.Parse(commandList[2] as it is repeated

Reply:
Also ban you use breaks to avoid else instate?

Reply:
Same for the cases below - formatting, try to use break, extract repeating parsing in variables